azkaban-web-server/src/main/java/azkaban/webapp/servlet/ProjectManagerServlet.java

```diff
@@ -1656,7 +1663,14 @@ private void ajaxHandleUpload(final HttpServletRequest req, final HttpServletRes
     }
 
     if (this.lockdownUploadProjects) {
-      registerError(ret, "Project uploading is locked out", resp, 400);
+      final String message = "Project uploading is locked out";
```

Comment:
See the earlier comment about the relationship between the lockdown config and the permissions.